qa/vagrant/src/test/resources/packaging/scripts/plugin_test_cases.bash

```diff
@@ -93,6 +93,12 @@ fi
     chown -R elasticsearch:elasticsearch "$ESPLUGINS"
 
     install_jvm_example
+    start_elasticsearch_service
+    # check that configuration was actually picked up    
+    curl -s localhost:9200/_cat/configured_example | sed 's/ *$//' > /tmp/installed
+    echo "foo" > /tmp/expected
+    diff /tmp/installed /tmp/expected
```

Comment:
This returns non-0 if these are different? I'm pretty sure it does but I just want to double check.